lib/pages/intro_screen.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+
+class IntroScreen extends StatelessWidget {
+  const IntroScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
+        appBar: AppBar(
+            leading: IconButton(
+                onPressed: () {},
+                icon: Icon(
+                  Icons.arrow_back_ios_sharp,
+                  color: Theme.of(context).iconTheme.color,
+                ))),
+        body: Container(
+          padding: const EdgeInsets.symmetric(horizontal: 50),
+          child: Center(
```

Comment:
no need of center if column alignment is already center

Reply:
ok done